src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -543,6 +556,15 @@ static void client_set_state(sd_dhcp6_client *client, DHCP6State state) {
                          dhcp6_state_to_string(client->state), dhcp6_state_to_string(state));
 
         client->state = state;
+
+        if (client->state_callback)
+                client->state_callback(client, state, client->state_userdata);
+}
+
+int sd_dhcp6_client_get_state(sd_dhcp6_client *client) {
+        assert(client);
+
+        return client->state;
```

Comment:
Please also expose the state enum in sd-dhcp6-client.h.

Reply:
Done.